### PR TITLE
Expand Bandwidth documentation

### PR DESCRIPTION
**Pull request by crayolakat**
## Description

Add more detailed Bandwidth documentation.

# Checklist:

- [ ] I have manually tested my changes on desktop and mobile
- [ ] The test suite passes locally with my changes
- [ ] If my change is a UI change, I have attached a screenshot to the description section of this pull request
- [X] [My change is 300 lines of code or less](https://github.com/MoveOnOrg/Spoke/blob/main/CONTRIBUTING.md#your-first-code-contribution), or has a documented reason in the description why it’s longer
- [X] I have made any necessary changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] My PR is labeled [WIP] if it is in progress

### CODE REVIEW

File: docs/HOWTO_INTEGRATE_BANDWIDTH.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## Spoke Instructions
+
+1. Navigate to the admin settings page of an organization.
+2. Under the "Bandwidth Config" section, enter your Bandwidth account ID, API username, and API password. The API user must have the `Configuration` role in Bandwidth.
+3. In the Advanced tab, fill out the sub-account, location, and application IDs.
+4. Click the "SAVE CREDENTIALS" button.
+
+
+## ngrok
```
Comment (Arique1104) [inline docs/HOWTO_INTEGRATE_BANDWIDTH.md]: For readability, should we add some vthis section "Local Development Instructions w/ ngrok?"

Comment (ibrand) [reply docs/HOWTO_INTEGRATE_BANDWIDTH.md]: I like @Arique1104 's idea here, what if we added a link in that intro text at the top that says something like:

> For setting up a development environment with Bandwidth, read this section in our local development guide.

And link to a header in the local dev guide with this info

**Review by ibrand — COMMENTED** [review]
One suggestion from Arique that I agree with, otherwise this looks good

**Comment by crayolakat** [comment]
@Arique1104 @ibrand Thanks! I actually changed the status of this PR to [WIP] since there's a few important configuration steps that I realized are missing. I'll take your suggestions into account when I update the PR

**Comment by crayolakat** [comment]
@Arique1104 @ibrand I updated `docs/HOWTO_INTEGRATE_BANDWIDTH.md` for the intro text to link to the ngrok instructions. I left the ngrok instructions in the same file, but after re-reading your comments, I'm wondering if you were saying to move the ngrok instructions to the `docs/HOWTO_DEVELOPMENT_LOCAL_SETUP.md` file?

**Comment by ibrand** [comment]
Yeah that was the suggestion, I think it's helpful to move all of the dev env instructions into one place

**Comment by crayolakat** [comment]
@ibrand Got it, pushed the relevant commits and also added some additional documentation details

**Review by ibrand — DISMISSED** [review]
Awesome, thank you for this write up!

**Comment by crayolakat** [comment]
This was merged into the `main` branch for the 12.2 release, but some rebasing caused the PR to not close automatically. Manually closing the PR.

**Pull request closed without merging**